src/main/java/net/minecraftforge/client/event/RenderNameplateEvent.java

```
@@ -0,0 +1,68 @@
+package net.minecraftforge.client.event;
+
+import com.google.common.base.Strings;
+import net.minecraft.entity.Entity;
+import net.minecraftforge.common.MinecraftForge;
+import net.minecraftforge.eventbus.api.Event;
+
+/**
+ * RenderNameplateEvent is fired whenever the entity renderer attempts to render a name plate/tag of an entity.
+ * <br>
+ * {@link #entity} contains the entity in which the plate is being rendered to. This cannot be changed by mods.<br>
+ * {@link #nameplateContent} contains the content being rendered on the name plate/tag. This can be changed by mods.<br>
+ * {@link #originalContent} contains the original content being rendered on the name plate/tag. This cannot be changed by mods.<br>
+ * <br>
+ * This event {@link HasResult}. <br>
+ * This event has a result. ALLOW will force-render name plate/tag, DEFAULT will ignore the hook and continue using the vanilla check
+ * & DENY will prevent name plate/tag from rendering<br>
+ * <br>
+ * This event is fired on the {@link MinecraftForge#EVENT_BUS}.
+ **/
+@Event.HasResult
+public class RenderNameplateEvent extends Event
+{
+
+    private final Entity entity;
+    private String nameplateContent;
+    private final String originalContent;
+
+    public RenderNameplateEvent(Entity entity, String content)
+    {
+        this.entity = entity;
+        this.setContent(content);
+        this.originalContent = Strings.nullToEmpty(content);
```

Why are we silently converting null to empty?